include/pybind11/pybind11.h

```diff
@@ -763,7 +763,10 @@ class cpp_function : public function {
         } else if (!result) {
             std::string msg = "Unable to convert function return value to a "
                               "Python type! The signature was\n\t";
-            msg += it->signature;
+            if (it == nullptr)
+                msg += "no signature found (nullptr)";
```

Comment:
Please change `msg` to say so; the full phrase

> The signature was
> no signature found

reads weird.

Reply:
makes sense, updated!